docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Spryker Cloud Commerce OS management
+description: Assignment of responsibilities, consultation and reporting obligations.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+This RACI Matrix defines responsibilities, consultation and reporting obligations while providing Platform-as-a-Service (PaaS) hosting for Spryker customers.
+
+<div class="bg-section">
```

Comment:
@andriitserkovnyi I've changes Glossary according to the structure you've shared